.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4.0"
+      }
+    }]
+  ],
+  "plugins": [
+    "add-module-exports",
```

Comment:
I don't think we need `add-module-exports`, since we only have named exports on package-level.